tests/test_search.py

```diff
@@ -4,12 +4,242 @@
 from util.solver_workflow import new_solver_session  # noqa: F401
 
 import ansys.fluent.core as pyfluent
-from ansys.fluent.core.utils.search import _get_version_path_prefix_from_obj
+from ansys.fluent.core.utils.search import (
+    _get_api_object_names,
+    _get_close_matches_for_word_from_names,
+    _get_version_path_prefix_from_obj,
+    _get_wildcard_matches_for_word_from_names,
+    _search,
+    _search_semantic,
+    _search_whole_word,
+    _search_wildcard,
+)
+
+
+def test_nltk_data_download():
+    import os
+    from pathlib import Path
+
+    import nltk
+    import platformdirs
+
+    packages = ["wordnet", "omw-1.4"]
+    for package in packages:
+        nltk.download(package, quiet=True)
+
+    with pytest.raises(LookupError):
+        _search_semantic("读", language="cmn")
+
+    user_data_path = Path(platformdirs.user_data_dir()).resolve()
+    nltk.data.path.append(user_data_path)
+    os.environ["NLTK_DATA"] = user_data_path
+    for package in packages:
+        nltk.download(package, download_dir=user_data_path, quiet=True)
+
+    with not pytest.raises(LookupError):
+        _search_semantic("读", language="cmn")
```

Comment:
We have observed that `nltk_data` is downloaded correctly in Windows and Linux but it not downloaded at default locations in GitHub runners so to validate this we have added this test.